logp/core_test.go

```diff
@@ -96,6 +96,36 @@ func TestLoggerLevel(t *testing.T) {
 	}
 }
 
+func TestLoggerSetLevel(t *testing.T) {
+	if err := DevelopmentSetup(ToObserverOutput()); err != nil {
+		t.Fatal(err)
```

Comment:
[Suggestion]
It helps to add a message explaining what failed
```suggestion
		t.Fatalf("cannot initialise logger on development mode: %s", err)
```